preprocessing-standards.Rmd

```diff
@@ -134,22 +138,23 @@ cwp <- CentWaveParam(ppm = 50,
 
 data <- findChromPeaks(data, param = cwp) 
```

Comment:
@andreavicini please run also `refineChromPeaks` after calling `findChromPeaks`. Use the settings from Anna's document for that.